js/scenes/Home.js

```diff
@@ -144,16 +154,26 @@ const styles = StyleSheet.create({
     ...defaultTextStyle,
   },
 
-  centering: {
+  loading: {
     alignItems: 'center',
     justifyContent: 'center',
     padding: 8,
   },
 });
 
-const mapStateToProps = ({ items }) => {
+const mapStateToProps = ({ items, itemsCache }) => {
+  const itemIds = items.value || [];
+
+  // create a list of fetched/inflated items
+  const fetchedItems = itemIds.reduce((result, id) => {
```

Comment:
filter and map sounds easier to understand to me.
If you think that in general reduce is OK for most people, just leave it as is.